api/payara-bom/pom.xml

```diff
@@ -441,12 +441,12 @@
             <dependency>
                 <groupId>org.glassfish.mq</groupId>
                 <artifactId>mq-distribution</artifactId>
-                <version>${mq.version}</version>
+                <version>${mq-distribution.version}</version>
```

Comment:
By my suggested "use project names" convention this should probably be `openmq.version`